receiver/nvmlreceiver/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+name: nvmlreceiver
+
+attributes:
+  gpu_number:
+    description: GPU index starting at 0.
+
+  memory_state:
+    description: GPU memory used or free.
+    enum: [used, free]
+
+  model:
+    description: GPU model
+
+metrics:
+  nvml.gpu.utilization:
+    enabled: true
+    description: Fraction of time GPU was not idle since the last sample.
+    unit: 10^2.%
+    gauge:
+      value_type: double
+    attributes: [model, gpu_number]
```

Comment:
I changed it to gpuIndex throughput. For the label it remains gpu_number because in the telemetry PRD we filed the reviewer stated he wanted it to be consistent with cpu_number for CPU utilization.